### ISSUE TITLE
Wrong background color detected by color contrast step

### ISSUE THREAD

**Issue by joelverhagen** [microsoft/accessibility-insights-web#692]
## Describe the bug

1. Go to https://www.nuget.org/.
1. Scroll down to the footer.
1. Run the Fast Pass mode.
1. See contrast errors and drill into the details of one (they all have the same problem)

> Element has insufficient color contrast of 1.2 (foreground color: #e3ebf1, background color: #ffffff, font size: 12.0pt, font weight: normal). Expected contrast ratio of 4.5:1

## Expected behavior

The actual background color is #002440, not #ffffff.

## Screenshots

![image](https://user-images.githubusercontent.com/94054/58194075-6a6e2080-7c79-11e9-9e68-c049ab7667aa.png)

## Extension (please complete the following information)

- OS: Windows
- Chrome Extension Version: 2.4.1
- Browser Version: Version 74.0.3729.157 (Official Build) (64-bit)
- Target Page: https://www.nuget.org/

## Are you willing to submit a PR?

Not sure. Depends on the complexity.

## Did you search for similar existing issues?

Yes, searched on GitHub. Found https://github.com/microsoft/accessibility-insights-web/issues/601 but it's not clear to me whether that is the same issue.

**Comment by dbjorge** [comment]
@lisli1 @peterdur @iamrafan and I took a look at this. We think we understand the root cause:

axe-core's implementation of the color contrast test identifies the "background color" at a location by querying `document.elementsFromPoint` at the location of the foreground element in question and considering the background-colors provided by the returned elements. Here, because the footer element in question that provides the background color has a `display: table-row` style, it is affected by [Chromium bug 647594](https://bugs.chromium.org/p/chromium/issues/detail?id=647594) and is incorrectly omitted from the `elementsFromPoint` output, so axe doesn't "see" it.

Unfortunately, it looks like the Chromium bug is a bit stagnant...

**Comment by joelverhagen** [comment]
Is there something simple we can do to our markup or CSS to work around the issue? For example can we apply the background color to another part of the DOM so it passes validation? This would future members of my team save effort reinvestigating the issue.

Thanks for the update.

**Comment by dbjorge** [comment]
Replicating the `background-color: #002440;` style from `.footer` in `.footer .container` would be a straightforward workaround, I think.

**Comment by WilcoFiers** [comment]
I reported on issue for this in axe-core here: https://github.com/dequelabs/axe-core/issues/1588

**Comment by joelverhagen** [comment]
I noticed that I am assigned. Is there something you would like me to check on?

**Comment by devaradhanm** [comment]
@joelverhagen , we would like to know if the workaround @dbjorge  provided unblocks you. Chrome team has changed the priority of the bug to P1. So, hopefully we will have this issue resolved soon. Do you have any questions? Feel free to close this issue if you are unblocked.

**Comment by joelverhagen** [comment]
Yes, the steps @dbjorge prescribed worked around the issue.

**Comment by devaradhanm** [comment]
Thanks for confirming @joelverhagen . I will close this issue then.
